spec/lib/tasks/bulk_import_documents_from_csv_spec.rb

```diff
@@ -37,33 +37,23 @@
 
 RSpec.describe "bulk_import_documents_from_csv", type: :task do
   let(:task) { Rake::Task["bulk_import_documents_from_csv"] }
-  let(:csv_path) { Rails.root.join("publications_export.csv") }
+  let(:csv_path) { "dummy_path.csv" }
 
   before(:each) do
     allow($stdout).to receive(:write)
+    allow(File).to receive(:exist?).with(csv_path).and_return(true)
   end
 
   after(:each) do
     task.reenable
-    File.delete(csv_path) if File.exist?(csv_path)
   end
 
   it "imports documents from a CSV file" do
-    CSV.open(csv_path, "w") do |csv|
-      csv << %w[
-        title
-        summary
-        body
-        attachment_title
-        attachment_filename
-        attachment_url
-        attachment_created_at
-        attachment_updated_at
-      ]
-      csv << [
-        "Design hearing decision: O/0567/25",
-        '"Outcome of request to invalidate, hearing held on 24 June 2025."',
-        '"| **Litigants** | Caesar Commerce Ltd v Huizhou New Road Cosmetics Company Limited |
+    csv_data = [
```

Comment:
I've no objection to the `Stub CSV IO in tests` changes but feel this should be 'fixed up' so that right from the start it is the approach taken for CSV testing.